src/java/org/apache/cassandra/hadoop/cql3/CqlConfigHelper.java

```diff
@@ -337,7 +340,12 @@ public static Cluster getCluster(String[] hosts, Configuration conf, int port)
                 .withQueryOptions(queryOptions)
                 .withPoolingOptions(poolingOptions);
 
-        return builder.build();
+        Cluster cluster = builder.build();
+
+        JmxReporter.forRegistry(cluster.getMetrics().getRegistry())
+                .inDomain(cluster.getClusterName() + "-metrics")
+                .build().start();
+        return cluster;
```

Comment:
flagging for later: potential leak of the jmx reporter here; we could make a subclass of Cluster that is a decorator and closes the reporter when the cluster is closed as well